js/utils/InitModules/InitOOJQuery.js

```diff
@@ -1,13 +1,14 @@
 require("./InitOO.js");
 
-if (!window.$)
-{
+if ($ !== undefined) {
+  $.noConflict();
+}
+if (!(jQuery && $)) {
```

Comment:
Not sure I understand why we check for both `jQuery` and `$`. Same with `OO.jQuery` and `OO.$`. Can we add some comments to explain the logic here?